samcli/local/lambda_service/service.py

```diff
@@ -0,0 +1,128 @@
+"""Local Lambda Service"""
+
+import logging
+import re
+import io
+import os
+
+from flask import Flask, request
+
+
+from samcli.local.services.base_service import BaseService
+from samcli.local.lambdafn.exceptions import FunctionNotFound
+
+LOG = logging.getLogger(__name__)
+
+
+class LocalLambdaService(BaseService):
```

Comment:
worth calling this something other than `LocalLambdaService`. "Service" isn't too clear on what it means